packages/qwik/src/core/render/jsx/types/jsx-generated.ts

```diff
@@ -1155,6 +1155,8 @@ export interface SVGAttributes<T> extends AriaAttributes, DOMAttributes<T> {
   xmlBase?: string | undefined;
   xmlLang?: string | undefined;
   xmlns?: string | undefined;
+  // TODO: proper xml namespacing support
+  'xmlns:xlink'?: string | undefined;
```

Comment:
I'd never heard of xml namespacing before, but it seems like actually you would want some type like ```
"xmlns" in Type ? { [`xmlns:${key}` for key in keyof SVGAttributes]: SVGAttributes[key] } : {}
``` and `&` it...or something...